dev/io.openliberty.pages.3.1.internal_fat/fat/src/io/openliberty/pages31/fat/tests/MiscTests.java

```diff
@@ -9,9 +9,10 @@
  *******************************************************************************/
```

Comment:
Can you give me some details why some asserts were removed?

Reply:
There were not removed on purpose. I've undone the removal.  Perhaps a merge conflict error on my part.